requirements.txt

```diff
@@ -1 +1,3 @@
+numpy
+pytest
```

Comment:
Why would users need pytest? This should only be for developers.
Requirements.txt is for users who do not wish to run tests and just want to use the package.
Requirements.dev.txt is for developers who need to be able to run tests.

I can understand the use of numpy for performance, but we don't want the dependencies of this package to grow unnecessarily.

Reply:
Sorry, that's all because of pycharm warnings and "convenient" automatic fixes. I also noticed this redundant addition of pytest, but forgot to remove it at final commit. Will fix this.